app/graphql/types/query_type.rb

```diff
@@ -33,12 +33,17 @@ def emergency_contact(id:)
       argument :id, ID, required: true
     end
 
-    def trips
```

Comment:
I understand why you removed this.  Good thinking, we can see all of a users trips with a User query.